#for joint-mecanum vehicles
#l:left wheel
#r:right wheel
#count clockwise,such as 1:front,2:rear or 1:front-left,2: front-right, 3:rear-right, 4:rear-left

VW_Center vw_center

#front agv
float64 V1
float64 V2
float64 V3
float64 V4

#rear agv
float64 V5
float64 V6
float64 V7
float64 V8
